third_party/move/move-compiler-v2/src/pipeline/explicit_drop.rs

```diff
@@ -136,61 +143,79 @@ impl<'a> ExplicitDropTransformer<'a> {
     }
 
     fn drop_temp(&mut self, tmp: TempIndex, attr_id: AttrId) {
```

Comment:
No we do not need this function, removed.

I agree this code could benefit from some refactoring, but it would be @fEst1ck 's call to make a 2nd pass. We all know more after the first round.